ui-particles-angular/projects/ui-particles-angular/src/lib/oem-components/gio-menu/gio-menu.stories.ts

```diff
@@ -24,7 +24,7 @@ import { GioMenuModule } from './gio-menu.module';
 import { GioMenuItemComponent } from './gio-menu-item/gio-menu-item.component';
 
 export default {
-  title: 'Components / Menu',
+  title: 'OEM Components / Menu',
```

Comment:
This would create another folder I believe. Should we consolidate 'OEM Components' and 'OEM Theme'?